qutebrowser/browser/downloads.py

```diff
@@ -695,12 +731,13 @@ def fetch(self, reply, fileobj=None, filename=None, auto_remove=False):
         """
         if fileobj is not None and filename is not None:
             raise TypeError("Only one of fileobj/filename may be given!")
-        if filename is not None:
-            suggested_filename = os.path.basename(filename)
-        elif fileobj is not None and getattr(fileobj, 'name', None):
-            suggested_filename = fileobj.name
-        else:
-            _inline, suggested_filename = http.parse_content_disposition(reply)
+        if not suggested_filename:
+            if filename is not None:
+                suggested_filename = os.path.basename(filename)
+            elif fileobj is not None and getattr(fileobj, 'name', None):
+                suggested_filename = fileobj.name
+            else:
+                _, suggested_filename = http.parse_content_disposition(reply)
```

Comment:
Awh, then the line will be to long. Is it ok to name it just `_in`, or should I rename the other argument to `suggested_fn`. Otherwise I really don't know how to indent that line.